engine/validation/fields.php

```diff
@@ -279,17 +279,19 @@ function check(string $msg = null)
     }
 
     /** Set upper date limit */
-    function before(string|null $date, string $msg = null)
+    function max(string|null $date, bool $addAttribute = true, string $msg = null)
```

Comment:
I would swap them around to prevent from having bugs :)
```suggestion
    function max(string|null $date, string $msg = null, bool $addAttribute = true)
```